### PR TITLE
in addition some session_finished_informations fixes

### PR DESCRIPTION
**Pull request by AleksanderKamienski**

### CODE REVIEW

File: evac/rvo2_dto.py [hunk 1]
```diff
@@ -92,34 +79,35 @@ def _find_closest_exit(self, e):
...
                     if exit['type'] == 'teleport':
                         evacuee.target_teleport_coordinates = (exit['direction_x'], exit['direction_y'])
                     if evacuee.target_teleport_coordinates is not None:
                         self.append_agents_to_move_downstairs(evacuee, e)
-                    paths_free_of_smoke.append([x, y, 0, exit])
+                    paths_free_of_smoke.append([x, y, 0, exit, path])
 
-                # paths.append([x, y, LineString(path).length])
             else:
                 paths.append([x, y, LineString(path).length, exit])
```
Comment (kowalskiw) [inline evac/rvo2_dto.py:92]: When the only path for agent is covered with smoke this list index  4 is out of range. Possibly the `path` should be added also to this list.

**Review by kowalskiw — CHANGES_REQUESTED** [review]
One addition in `evac.rvo2_dto_find_closest_exit()` required.

File: evac/rvo2_dto.py (hunk 1 above)
Comment (AleksanderKamienski) [reply evac/rvo2_dto.py]: I have already corrected a mistake that should not have happened

**Review by kowalskiw — APPROVED** [review]
All seems OK now - to be squashed&merged

**Pull request merged**